### PR TITLE
Adding a section dealing with ransomware protection

### PR DESCRIPTION
**Pull request by beniaju2**

### CODE REVIEW

**Comment by Jak-MS** [comment]
@chcomley 
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Review by Jominana — APPROVED** [review]

**Review by chcomley — APPROVED** [review]

**Comment by chcomley** [comment]
#sign-off

**Comment by chcomley** [comment]
#sign-off

**Review by chcomley — APPROVED** [review]

**Comment by chcomley** [comment]
#sign-off

**Comment by chcomley** [comment]
@Court72 Hello, are you able to merge this please?

File: docs/organizations/security/data-protection.md [hunk 1]
```diff
@@ -182,6 +182,10 @@ Where a possible intrusion or high priority security vulnerability gets identifi
 
 Finally, to help combat emerging threats, Azure DevOps employs an "Assume Breach" strategy. A highly specialized group of security experts within Microsoft, known as the Red Team, assumes the role of sophisticated adversaries. This team tests breach detection and response, to accurately measure readiness and the impacts of real-world attacks. This strategy strengthens threat detection, response, and defense of the service. It also allows the team to validate and improve the effectiveness of the entire security program.
 
+### Ransomware Attack Protection Overview
+
+Azure DevOps is built on Azure and leverages Azure's best in class controls to help prevent, detect and/or respond to a ranswomware attack. For more information on how Azure protects customers from Ransomware attacks, please review the [Azure Defenses for Ransomware Attack whitepaper](https://azure.microsoft.com/resources/azure-defenses-for-ransomware-attack/).
+
```
Comment (ShannonLeavitt) [inline docs/organizations/security/data-protection.md:188]:
```suggestion
### Ransomware attack protection overview

Azure DevOps is built on Azure and uses Azure's best-in-class controls to help prevent, detect, and respond to a ransomware attack. For more information about how Azure protects customers from ransomware attacks, see [Azure defenses for ransomware attack](https://azure.microsoft.com/resources/azure-defenses-for-ransomware-attack/).

```
Fixing typo and editing for Microsoft style

**Review by chcomley — APPROVED** [review]

**Comment by chcomley** [comment]
#sign-off

**Pull request merged**